homeassistant/components/x10/light.py

```diff
@@ -34,7 +34,7 @@ def x10_command(command):
 
 def get_unit_status(code):
     """Get on/off status for given unit."""
-    output = check_output(f"heyu onstate {code}", shell=True)
+    output = check_output(["heyu", "onstate", code])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4852890)